script/sdbSpecList.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+#
+# This shell script prints tizen device serial name on each line.
+#
+# on `sdb devices`, it will show result below.
+#
+# List of devices attached 
+# 192.168.223.1:26101 	device    	0
+# 192.168.224.1:26101 	device    	0
+#
+# so, only imformation we need is list of -s in first column from second row to end.
+# just pass `sdb device` result except first line and grep only device and pring first column
```

Comment:
- I am not sure what `pring` is. `print`??
- Please use correct English grammar...
  - First word in statement starts with capital letter.
  - Please put `.` at the end of statement. (L12?). Since L13 does not ends with `.`, its hard to tell if L13 and L14 is one statement or two statements)
  - Please replace first `and` to `,`